geonotebook/utils.py

```diff
@@ -13,3 +13,6 @@ def get_kernel_id(kernel):
     connection_file_path = ipykernel.get_connection_file()
     connection_file = os.path.basename(connection_file_path)
     return connection_file.split('-', 1)[1].split('.')[0]
+
+def transform_coordinates(source_srs, target_srs, x, y):
```

Comment:
At first that is what I did. Then I realized it would be a better utility function with the ability to use it the other way around(Converting to wgs84). It takes an array of coordinates now.